parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -198,28 +198,29 @@ downloadContainerdWasmShims() {
     for shim_version in $CONTAINERD_WASM_VERSIONS; do
         binary_version="$(echo "${shim_version}" | tr . -)"
         local containerd_wasm_filepath="/usr/local/bin"
-        BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER="${BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER:=}"
```

Comment:
is this removing support to download the shims via oras? is that something we agreed on already?

@AlisonB319

Reply:
Yeah we don't want to be removing oras download of wasm if these new versions can be installed during provisioning. @Mossaka is there a reason you removed this code path?

context - oras is needed to download a package when it is trying to be installed during node provisioning (aka not cached during build time).

Reply:
Ah I removed this by accident, will add it back. Good catch!

Reply:
Does `BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER/oss/binaries/` have the binaries that released by the [containerd-wasm-shims pipeline](https://dev.azure.com/AzureContainerUpstream/Kubernetes/_build?definitionId=499)?

Reply:
if we're gonna support v015.1 then we should have the corresponding tag in MCR, how can we go about adding it?

Reply:
I asked @Mossaka to reach out to Xinhe Li about adding v0.15.1. But I am unsure if 0.15.1 should block this PR since NIC is not in place yet.

Reply:
synced some more offline, sounds like this will be needed shortly (~2 weeks) to ensure NIC bootstrapping is stable, @Mossaka I'd prefer we get this resolved before merging this

Reply:
the MCR changes are completely out of my radar. I know @fseldow's been working on it, but not sure when it will have that in MCR.

Reply:
Would it be possible to notify me or my team in the future for any changes that made to the Wasm shims? @AlisonB319

Reply:
sounds good to me! Happy to review your PR.

Reply:
Gonna resolve this comment. See [my comment](https://github.com/Azure/AgentBaker/pull/4744#issuecomment-2359381086) below